src/main/java/org/example/c06challenges/MaxNumberCountInArray.java

```diff
@@ -0,0 +1,36 @@
+package org.example.c06challenges;
+
+public class MaxNumberCountInArray {
+    private MaxNumberCountInArray() {
+    }
+
+    public static int majorityElementOfArray(int[] array) {
+        int arrayLength = array.length;
+        int max_count = 0;
+        int maxFreqNumber = -1;
+
+        if (arrayLength == 0) {
+            return -2;
+        } else {
+
+            for (int i = 0; i < arrayLength; i++) {
+                int count = 0;
+                for (int j = 0; j < arrayLength; j++) {
+
+                    if (array[i] == array[j]) {
+                        count++;
+                    }
+                }
+                if (count > max_count) {
+                    max_count = count;
+                    maxFreqNumber = array[i];
+                }
+
+                if(max_count < 2){
```

Comment:
This `if` is not necessary. Just let the solution return the first most frequent element in the array.